test/Test/Pos/Ssc/GodTossing/ComputeSharesSpec.hs

```diff
@@ -0,0 +1,40 @@
+-- | Specification of Pos.Ssc.GodTossing.Toss.Base.computeSharesdistr
+
+module Test.Pos.Ssc.GodTossing.ComputeSharesSpec
+       ( spec
+       ) where
+
+import           Test.Hspec              (Spec, describe, it)
+import           Test.Hspec.QuickCheck   (prop)
+import           Universum
+
+import           Control.Monad.Except    (runExceptT)
+import qualified Data.HashMap.Strict     as HM (elems, keys)
+import           Pos.Lrc                 (RichmenStake)
+import qualified Pos.Ssc.GodTossing      as T
+import           Pos.Types               (sumCoins)
+import           Test.QuickCheck         (Property, (===), (==>))
+import           Test.QuickCheck.Monadic (assert, monadicIO)
+
+spec :: Spec
+spec = describe "computeSharesDistr" $ do
+    prop emptyRichmenStakeDesc emptyRichmenStake
+    prop allRichmenGetShareDesc allRichmenGetShares
+  where
+    emptyRichmenStakeDesc = "Outputs an empty distribution when the richmen stake is\
```

Comment:
You're right, this isn't the first thing I wrote, but I forgot to change the definition after modifying the test. Thanks.